Sources/LCP/LCPAcquisition.swift

```diff
@@ -10,7 +10,7 @@ import R2Shared
 /// Represents an on-going LCP acquisition task.
 ///
 /// You can cancel the on-going download with `acquisition.cancel()`.
-public final class LCPAcquisition: Loggable, Cancellable {
+public final class LCPAcquisition: Loggable {
```

Comment:
I'm not sure if anything in LCP needs to change, if we get errors based on how the other packages are using it, we can keep the changes but we've never made any changes to this Pod as a Learn requirement